RFC/Proposals/readme.md

```diff
@@ -0,0 +1,13 @@
+## Proposals
+
+The "Proposals" directory is where you can find all the proposed changes and additions to the Dgraph Core and others. It
+contains the Request for Comment (RFC) documents. It is where anyone can submit their ideas for improving the Dgraph
+Core, whether it is through discussion on the forum or through independent research and development. If you have
+knowledge of the Core code and sufficient understanding of the problem that you're trying to solve, you can submit your
```

Comment:
this is line is not needed I think

Reply:
Sorry, I didn't understand. Which line?

Reply:
"If you have knowledge of the Core code and sufficient understanding of the problem that you're trying to solve, you can submit your..."